src/include/sof/audio/codec_adapter/codec/waves.h

```diff
@@ -0,0 +1,46 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2020 Waves Audio Ltd. All rights reserved.
+ *
+ * Author: Oleksandr Strelchenko <oleksandr.strelchenko@waves.com>
+ */
+#ifndef __SOF_AUDIO_WAVES_CODEC__
+#define __SOF_AUDIO_WAVES_CODEC__
+
+#include "MaxxEffect/MaxxEffect.h"
+#include "MaxxEffect/MaxxStream.h"
+#include "MaxxEffect/MaxxStatus.h"
+#include "MaxxEffect/Initialize/MaxxEffect_Initialize.h"
+#include "MaxxEffect/Process/MaxxEffect_Process.h"
+#include "MaxxEffect/Process/MaxxEffect_Reset.h"
+#include "MaxxEffect/Control/RPC/MaxxEffect_RPC_Server.h"
+#include "MaxxEffect/Control/Direct/MaxxEffect_Revision.h"
+
+struct waves_codec_data {
```

Comment:
codec data is private to the codec, please move to the c file

Reply:
fixed